packages/console/src/pages/Guide/index.tsx

```diff
@@ -1,51 +1,76 @@
 import { AdminConsoleKey } from '@logto/phrases';
-import React, { cloneElement, isValidElement, PropsWithChildren, ReactNode, useState } from 'react';
+import { MDXProvider } from '@mdx-js/react';
+import i18next from 'i18next';
+import { MDXProps } from 'mdx/types';
+import React, {
+  cloneElement,
+  isValidElement,
+  lazy,
+  LazyExoticComponent,
+  PropsWithChildren,
+  ReactNode,
+  Suspense,
+  useState,
+} from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
-import ReactMarkdown from 'react-markdown';
 
 import Button from '@/components/Button';
 import CardTitle from '@/components/CardTitle';
+import CodeEditor from '@/components/CodeEditor';
 import DangerousRaw from '@/components/DangerousRaw';
 import IconButton from '@/components/IconButton';
 import Spacer from '@/components/Spacer';
 import Close from '@/icons/Close';
-import { GetStartedForm } from '@/types/get-started';
+import { GuideForm } from '@/types/guide';
 
+import MultiTextInputField from './components/MultiTextInputField';
 import Step from './components/Step';
-import { GetStartedType, useGetStartedSteps } from './hooks';
 import * as styles from './index.module.scss';
 
+const Guides: Record<string, LazyExoticComponent<(props: MDXProps) => JSX.Element>> = {
+  react: lazy(async () => import('@/assets/docs/tutorial/integrate-sdk/react/index.mdx')),
+  'react_zh-cn': lazy(
```

Comment:
will use `en` as fallback?

Reply:
Yes, those without a locale suffix is the english version. And in line 71 there is:
```typescript
const guideKey = `${subtype}_${locale}`.toLowerCase();
const GuideComponent = Guides[guideKey] ?? Guides[subtype];
```
This will fallback to the english version (no local suffix) if it fails to find a version that matches current locale.

Reply:
oh shit i missed this sry

Reply:
No worries